webui/react/src/pages/TaskList.tsx

```diff
@@ -58,178 +57,34 @@ interface SourceInfo {
   sources: TensorBoardSource[];
 }
 
-const defaultFilters: TaskFilters<CommandType> = {
-  limit: MINIMUM_PAGE_SIZE,
-  states: undefined,
-  types: undefined,
-  users: undefined,
-};
-
-const defaultSorter: ApiSorter = {
-  descend: true,
-  key: 'startTime',
-};
-
-const URL_ALL = 'all';
-
-const STORAGE_PATH = 'task-list';
-const STORAGE_FILTERS_KEY = 'filters';
-const STORAGE_LIMIT_KEY = 'limit';
-const STORAGE_SORTER_KEY = 'sorter';
+const filterKeys: Array<keyof Settings> = [ 'search', 'state', 'type', 'user' ];
+const settingsBasePath = paths.taskList();
```

Comment:
ditto https://github.com/determined-ai/determined/pull/2786#discussion_r681208872

Reply:
did you mean to use the `storagePath` from `..settings.ts` file?

Reply:
the `storagePath` is meant for the local storage. Was thinking at one point of doing the same for `settingsBasePath` but on pages with variables in the route paths (like experiment detail `/experiment/<id>`) it would require a bit of processing.